src/shapes/Path.ts

```diff
@@ -77,14 +77,31 @@ export class Path<
   ) {
     super(options as Props);
     const pathTL = this._setPath(path || []);
-    const origin = this.translateToGivenOrigin(
-      new Point(left ?? pathTL.x, top ?? pathTL.y),
-      typeof left === 'number' ? this.originX : 'left',
-      typeof top === 'number' ? this.originY : 'top',
-      this.originX,
-      this.originY
-    );
-    this.setPositionByOrigin(origin, this.originX, this.originY);
+    if (this.fromSVG) {
+      // if coming from SVG just assign top and left, those will be fixed
+      // later by remove removeTransformMatrixForSvgParsing using the weird
+      // _findCenterFromElement.
+      // This is not clear and is probably wrong, but as of now it works in conjuction
+      // with that other parsing function
+      this.set({ left: pathTL.x, top: pathTL.y });
```

Comment:
I don't like this.
Can we instead handle the logic in `fromElement` and set the origin to left/top. Then prev logic should work